beekeeper-core/src/main/java/com/expediagroup/beekeeper/core/context/MonitoringContext.java

```diff
@@ -32,7 +31,7 @@
 public class MonitoringContext {
 
   @Bean
-  MeterRegistry registry(GraphiteConfigFactory graphiteConfigFactory) {
+  GraphiteMeterRegistry registry(GraphiteConfigFactory graphiteConfigFactory) {
```

Comment:
I remember us having to do something similar in WaggleDance for the same reason (see https://github.com/HotelsDotCom/waggle-dance/blob/master/waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/metrics/MonitoringConfiguration.java). There we also renamed the method to be `graphiteMeterRegistry` - not sure what (if any) difference that makes.

Reply:
Yep spotted that in WD. Not sure if it makes a difference, but will rename for consistency.

Reply:
It does make a (small) difference; the method name affects the bean name, which is used somewhere else. All good.